dataloader/preprocessor.hpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file dataloader.hpp
+ * @author Kartik Dutt
+ * 
+ * Definition of PreProcessor class for popular datasets.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MODELS_PREPROCESSOR_HPP
+#define MODELS_PREPROCESSOR_HPP
+
+#include <mlpack/prereqs.hpp>
+
+/**
+ * Contains standatd pre-process functions for popular datasets.
+ *
+ * @tparam DatasetX Datatype for loading input features.
+ * @tparam DatasetY Datatype for prediction features.
+ */
+template<
+  typename DatasetX = arma::mat,
+  typename DatasetY = arma::mat
+>
+class PreProcessor
+{
+ public:
+  static void MNIST(DatasetX& /*trainX*/, DatasetY& trainY,
```

Comment:
```suggestion
  static void MNIST(DatasetX& /* trainX */, DatasetY& trainY,
```

Just to be consistent with the rest.